### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

- makes form share dialog close button , and heading more like other dialog boxes
- makes formio button component match other buttons on app
- make all disabled form fields in submission review mode have the same grey background

the 'manage', 'view submissions', 'launch' etc links on my forms table have a strange underline on hover.
can't for life of me fix that. might have to put in a different ticket

i've also found that my npm build times (triggered by hot-reloading) were getting quite slow. Not sure if it's because of my inefficient or greedy css rules. it seems the @extend rule slowed things down especially.
however, after rebooting my pc and now it doesnt seem slow.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-73

**Review by loneil — APPROVED** [review]
👍

File: app/frontend/src/assets/scss/style.scss [hunk 1]
```diff
@@ -298,13 +298,15 @@ a,
...
   }
+
   // formio submit buttons
-  .formio-component-submit .btn {
+  .formio-component-button .btn {
     text-transform: uppercase;
+    padding: 8px 16px;
+    line-height: 21px;
+    font-weight: 700;
   }
```
Comment (parc-jason) [inline app/frontend/src/assets/scss/style.scss:308]: Don't know if this is design system or vuetify, but also:
```
letter-spacing: .0892857143em;
font-size: .875rem;
```

**Review by parc-jason — APPROVED** [review]
Approving with a couple of enhancements.
Looks great!

**Pull request merged**